hack/verify-govet.sh

```diff
@@ -58,7 +58,7 @@ for i in $targets
     #
     # The intended result is that each incantation of this line returns
     # either 0 (pass) or 1 (fail).
-    godep go vet "${goflags[@]:+${goflags[@]}}" "$i" 2>&1 \
+    go vet "${goflags[@]:+${goflags[@]}}" "$i" 2>&1 \
```

Comment:
On line 42 above, you might want to add something to filter out vendor since `go list` will list vendored packages (unlike Godeps/_workspace).